editor/blocks/quote-block/index.js

```diff
@@ -0,0 +1,23 @@
+const { query, html, text } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'wp/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: query( 'cite,footer', text() )
```

Comment:
There's a difference. `footer` is a block element, which should be the wrapper of `cite` (if any). `cite` can also be used elsewhere, like in `figcaption`.